ui/pages/confirmations/components/simulation-details/amount-pill.tsx

```diff
@@ -53,10 +54,18 @@ export const AmountPill: React.FC<{
   }
 
   if (asset.tokenId) {
-    const tokenIdPart = `#${hexToDecimal(asset.tokenId)}`;
+    const decimalTokenId = hexToDecimal(asset.tokenId);
+    const shortenedDecimalTokenId = shortenAssetId(decimalTokenId, 11, 4, 4);
```

Comment:
These numbers would have more readability if we use object arguments as suggested.

Reply:
Good call done in https://github.com/MetaMask/metamask-extension/pull/25252/commits/7ae1d604fc3c9c559b2c4f32b62ad0e085184ea1